README.md

```diff
@@ -29,6 +29,20 @@ $ docker-compose exec app /bin/bash
 $ glide install
 ```
 
+### Setup project
```

Comment:
Chỉ cần 1 câu lệnh `-exec setup` thôi
detail thì `-exec setup --file /path/to/template/file`